wallets/dashcore/privatesend-instantsend.rst

```diff
@@ -152,6 +149,31 @@ created by the mixing process on the **Transactions** tab.
 
    Transactions created by PrivateSend on the Transactions tab
 
+The following table describes the PrivateSend-related transactions displayed in
+the Type column of the **Transactions** tab:
+
++------------------------------------+-------------------------------------------+
+| PrivateSend Transaction Type       | Transaction Description                   |
++====================================+===========================================+
+| PrivateSend Make Collateral Inputs | Wallet funds were moved to collateral     |
+|                                    | inputs that will be used to make          |
+|                                    | collateral payments. This is done to      |
+|                                    | minimize traceability of collaterals.     |
++------------------------------------+-------------------------------------------+
+| PrivateSend Create Denominations   | Wallet funds were broken into             |
+|                                    | PrivateSend denominations (Step 1         |
+|                                    | :ref:`here<privatesend>`)                 |
++------------------------------------+-------------------------------------------+
+| PrivateSend Denominate             | A transaction was sent to a masternode    |
+|                                    | in order to participate in a mixing       |
+|                                    | session (Step 3 :ref:`here<privatesend>`) |
++------------------------------------+-------------------------------------------+
+| PrivateSend Collateral Payment     | The mixing session collateral was         |
+|                                    | claimed. This fee is charged in ~10%      |
+|                                    | of mixing sessions to prevent spam        |
+|                                    | attacks.                                  |
++------------------------------------+-------------------------------------------+
```

Comment:
Btw, there is also `PrivateSend` tx type which is displayed for PrivateSend payments i.e. when you use mixed funds to pay someone. Not sure if it has to be in the table too or not though... Otherwise, looks good imo 👍

Reply:
Good point. I will add that one too.